### PR TITLE
Add [rotationQuaternion]

### PR DESCRIPTION
**Pull request by 3DpoNCh1k**

### CODE REVIEW

File: Quaternion.mpl [hunk 1]
```diff
@@ -192,3 +195,28 @@ axisAngleQuaternion: [
...
   ) quaternion
 ];
+
+rotationQuaternion: [
+  rotation:;
+
+  rotation 1.0e-12 rotationQuaternion
+];
+
+rotationQuaternion: [rotation: threshold:;; TRUE] [
```
Comment (ValeryAntopol) [inline Quaternion.mpl:205]: This overload checks only stack depth and will break if user wants to use 1-arg version when he has more values on uncontrolled stack. I suggest checking that `threshold` is a floating point number.

File: Quaternion.mpl [hunk 2]
```diff
@@ -192,3 +195,28 @@ axisAngleQuaternion: [
...
+  ] [
+    length: rotation length;
+    halfAngle: length 2.0 length cast /;
+    coefficient: halfAngle sin length /;
+    (
+      0 rotation @ coefficient *
+      1 rotation @ coefficient *
+      2 rotation @ coefficient *
+      halfAngle cos
+    ) quaternion
```
Comment (ValeryAntopol) [inline Quaternion.mpl:220]: This part is done by the `axisAngleQuaternion` above. I suggest reusing the code from this function.

**Review by ValeryAntopol — CHANGES_REQUESTED** [review]

**Review by ValeryAntopol — APPROVED** [review]
LGTM

File: Quaternion.mpl [hunk 3]
```diff
@@ -192,3 +198,21 @@ axisAngleQuaternion: [
     ah cos
   ) quaternion
 ];
+
+rotationQuaternion: [
+  rotation:;
+
```
Comment (MatwayBurkow) [inline Quaternion.mpl:204]: Redundant newline

**Review by Parean — APPROVED** [review]

**Pull request merged**